docs/writing/posts/learning.md

```diff
@@ -0,0 +1,153 @@
+---
+draft: False
+date: 2024-01-08
+slug: what-i-learned-from-trying
+categories:
+  - Personal
+authors:
+  - jxnl
+---
+
+# What I Learned from Trying
+
+After writing my post [advice for young people](./advice.md), a couple of people asked about my learning process. I could discuss overcoming plateaus or developing mastery, learning for the joy of learning. I could also talk about how to avoid feeling overwhelmed by new topics and break them down into smaller pieces. However, I think that has been done before.
+
+Instead, I'm going to explore a new style. I'm just going to go through a chronological telling of my life and what i learned about learning.
+
+I'm going to break this down into two sections:
+
+**What I did in high school and college**
+
+1. Design
+2. Data Science
```

Comment:
Maybe keep the bulletting style consistent between this and the after college list? Like both 1. 2. 3. or both - .